src/Sql/Combine.php

```diff
@@ -54,12 +58,11 @@ public function __construct($select = null, $type = self::COMBINE_UNION, $modifi
      * @param Select|array $select
      * @param string $type
      * @param string $modifier
-     *
-     * @return self Provides a fluent interface
+     * @return self
```

Comment:
IMHO it should be `$this`, as we are returning exactly the same instance.
I always refer to point 14-16 in the very bottom of PSR-5 draft: https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc.md#keyword